experimental/packages/sdk-logs/test/common/Logger.test.ts

```diff
@@ -18,8 +18,8 @@ import * as assert from 'assert';
 import * as sinon from 'sinon';
 
 import { LogRecord, Logger, LoggerConfig, LoggerProvider } from '../../src';
-import { loadDefaultConfig } from '../../src/config';
-import { context } from '@opentelemetry/api';
+import { ROOT_CONTEXT, TraceFlags, context, trace } from '@opentelemetry/api';
+import * as logsAPI from '@opentelemetry/api-logs';
```

Comment:
We avoid `import * as` for various reasons including minification and tree shaking